### PR TITLE
[ZT] posture logs

### PR DESCRIPTION
**Pull request by ranbel**
PCX-5892

### CODE REVIEW

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+|-------------|-------------|
+| **Email**   | Email used to register the device with Zero Trust            |
+| **User ID** | UUID of the user who registered the device           |
+
+### Posture details
+
+| Field               | Description |
+|---------------------|-------------|
+| **Name**            | Name of the [device posture check](/cloudflare-one/identity/devices) |
+| **Type**            | Type of [WARP client check](/cloudflare-one/identity/devices/warp-client-checks/) or [service provider check](/cloudflare-one/identity/devices/service-providers/).      |
```
Comment (maxvp) [inline content/cloudflare-one/analytics/logs/posture-logs.md:39]:
```suggestion
| **Type**            | Type of [WARP client check](/cloudflare-one/identity/devices/warp-client-checks/) or [service provider check](/cloudflare-one/identity/devices/service-providers/)      |
```

Comment (maxvp) [reply content/cloudflare-one/analytics/logs/posture-logs.md:39]: Should this column all have periods to keep consistency with `Conditions met`?

**Review by maxvp — APPROVED** [review]

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+| Field       | Description |
+|-------------|-------------|
+| **Email**   | Email used to register the device with Zero Trust            |
+| **User ID** | UUID of the user who registered the device           |
+
+### Posture details
+
+| Field               | Description |
+|---------------------|-------------|
+| **Name**            | Name of the [device posture check](/cloudflare-one/identity/devices) |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Name**            | Name of the [device posture check](/cloudflare-one/identity/devices). |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+| **Type**            | Type of [WARP client check](/cloudflare-one/identity/devices/warp-client-checks/) or [service provider check](/cloudflare-one/identity/devices/service-providers/).      |
+| **Rule ID**         | UUID of the device posture check            |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Rule ID**         | UUID of the device posture check.           |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+| **Conditions met**  | Whether the device passed or failed the posture check criteria. Evaluates to `true` if the **Received values** match the **Expected values**.      |
+| **Expected values** | Values required to pass the device posture check       |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Expected values** | Values required to pass the device posture check.       |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+| **Received values** | Posture check values detected by the WARP client  |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Received values** | Posture check values detected by the WARP client.  |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+| **Email**   | Email used to register the device with Zero Trust            |
+| **User ID** | UUID of the user who registered the device           |
+
+### Posture details
+
+| Field               | Description |
+|---------------------|-------------|
+| **Name**            | Name of the [device posture check](/cloudflare-one/identity/devices). |
+| **Type**            | Type of [WARP client check](/cloudflare-one/identity/devices/warp-client-checks/) or [service provider check](/cloudflare-one/identity/devices/service-providers/).      |
+| **Rule ID**         | UUID of the device posture check            |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Rule ID**         | UUID of the device posture check.           |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+Enterprise users can generate more detailed logs with [Logpush](/cloudflare-one/analytics/logs/logpush/).
+
+## Explanation of the fields
+
+### Device details
+
+| Field             | Description |
+|-------------------|-------------|
+| **Name**          | Name of the device  |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Name**          | Name of the device.  |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+| **Serial number** | Serial number of the device            |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Serial number** | Serial number of the device.           |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+| **Manufacturer**  | Manufacturer of the device            |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Manufacturer**  | Manufacturer of the device.            |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+| **Model**         | Model of the device  |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Model**         | Model of the device.  |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+### User details
+
+| Field       | Description |
+|-------------|-------------|
+| **Email**   | Email used to register the device with Zero Trust            |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **Email**   | Email used to register the device with Zero Trust.            |
```

File: content/cloudflare-one/analytics/logs/posture-logs.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+| **User ID** | UUID of the user who registered the device           |
```
Comment (ranbel) [inline content/cloudflare-one/analytics/logs/posture-logs.md]:
```suggestion
| **User ID** | UUID of the user who registered the device.           |
```

**Pull request merged**